src/variant.h

```diff
@@ -110,7 +110,7 @@ struct Variant {
   bool duckGating = false;
   bool staticGating = false;
   bool pastGating = false;
-  Bitboard staticGatingRegion = AllSquares;
+  Bitboard staticGatingRegion[COLOR_NB] = {AllSquares};
```

Comment:
I know it is just for testing, but just FYI this way of initialization does not work. If you initialize `= {0}` that will set all members to zero, but not because this is a general "fill", but because all remaining values will be initialized to the default/zero value. So this is equivalent to `{AllSquares, 0}`, and not `{AllSquares, AllSquares}` if that was the intention.